### PR TITLE
Fixed wrong order imports accordance to PEP

### PR DESCRIPTION
**Pull request by aloks98**
__init__.py : Fixed the import orders in __init__.py according to PEP8 Standards.

Fixes issue #2617

### CODE REVIEW

**Comment by mroutis** [comment]
Hello, @aloks98, thanks for the PR! Is this the only file with import order errors?

**Comment by aloks98** [comment]
No, there are more. I will be opening issues as I am looking through the code.

**Comment by mroutis** [comment]
Cool!

**Comment by mroutis** [comment]
By the way, are you fixing this errors manually or using some sort of PEP compliance tool?

**Comment by aloks98** [comment]
I am using pylint for checking the code, but fixing it manually

File: dvc/__init__.py [hunk 1]
```diff
@@ -5,9 +5,9 @@
 """
 
 from __future__ import unicode_literals
-
-from dvc.version import __version__  # noqa: F401
 import warnings
+from dvc.version import __version__  # noqa: F401
+
```
Comment (efiop) [inline dvc/__init__.py]: [Per PEP](https://www.python.org/dev/peps/pep-0008/#id23):

> You should put a blank line between each group of imports.

So please move `from dvc.version import __version__  # noqa: F401` underneath `import dvc.logger`.

File: dvc/__init__.py [hunk 2]
```diff
@@ -6,10 +6,11 @@
 
 from __future__ import unicode_literals
 
-from dvc.version import __version__  # noqa: F401
 import warnings
 import dvc.logger
```
Comment (efiop) [inline dvc/__init__.py]:
```suggestion

import dvc.logger
```

**Pull request merged**